RebootServer.py

```diff
@@ -167,7 +167,7 @@ def RunServer(miners):
             sleep(300)
 
             print("Starting regular checks at %s..." % getTime(), flush=True)
-            if connection_check==true:
+            if connection_check()==True:
```

Comment:
Just to make things a bit more Pythonic, could you make this `if connection_check():`